homepage/tests/appUser_model_tests.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from homepage.models import AppUser
+
+
+@pytest.mark.django_db
+class TestAppUser:
+
+    @pytest.fixture
+    def appUsers(self):
+        appUser1 = AppUser.create_appUser('user1', 'user1@mta.ac.il', 'useruser222')

```

Comment:
please fix all the occurrences according to:
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables
```suggestion
        app_user1 = AppUser.create_appUser('user1', 'user1@mta.ac.il', 'useruser222')

```

Reply:
fixed